core/server/worker/src/main/java/alluxio/worker/netty/DataServerShortCircuitWriteHandler.java

```diff
@@ -88,11 +98,12 @@ private void handleBlockCreateRequest(final ChannelHandlerContext ctx,
       @Override
       public Void call() throws Exception {
         if (request.getOnlyReserveSpace()) {
-          mBlockWorker.requestSpace(request.getSessionId(), request.getBlockId(),
+          mBlockWorker.requestSpace(mSessionId, request.getBlockId(),
               request.getSpaceToReserve());
           ctx.writeAndFlush(RPCProtoMessage.createOkResponse(null));
         } else {
-          String path = mBlockWorker.createBlock(request.getSessionId(), request.getBlockId(),
+          mSessionId = IdUtils.getRandomNonNegativeLong();
+          String path = mBlockWorker.createBlock(mSessionId, request.getBlockId(),
```

Comment:
Any risk of getting a second create call and orphaning the previous session? Should handle this similar to the lock id?